public/manifest.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
so freaking cool that you figured this stuff out!!!

Reply:
did you have a look at service workers as well for offline supporting your app?

Reply:
https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Offline_Service_workers

Reply:
Yeah, but the whole redirecting thing seemed to be a nightmare with caching